src/libs/Fullstory/index.native.ts

```diff
@@ -1,13 +1,32 @@
 import FullStory, {FSPage} from '@fullstory/react-native';
-import type {OnyxEntry} from 'react-native-onyx';
+import {type OnyxEntry} from 'react-native-onyx';
```

Comment:
```suggestion
import type {OnyxEntry} from 'react-native-onyx';
```

This seems to fix the error for me. Does it cause a problem?